lib/modules/datasource/index.ts

```diff
@@ -415,6 +416,60 @@ export async function getPkgReleases(
             version.matches(constraintValue, releaseConstraint)
         );
       });
+    } else {
+      if (constraintName === 'php') {
```

Comment:
Ideally don't hardcode this so that it can be reused for other ecosystems in future without a rewrite (especially python and node)

Reply:
While skimming through the code yesterday I thought the filtering could be done directly in the packagist lookup. Was I wrong or did you out it here to be potentially reusable on purpose?